Docker/README.md

```diff
@@ -351,3 +351,27 @@ server {
 	}
 }
 ```
+### Alternative reverse proxy using [apache 2.4](https://httpd.apache.org/docs/2.4/howto/reverse_proxy.html)
+
+Here is an example of configuration to run FreshRSS behind an Apache reverse proxy (as subdirectory).
```

Comment:
```suggestion
Here is an example of a configuration file for running FreshRSS behind an Apache reverse proxy (as a subdirectory).
```

But it can probably lose a few words:

```suggestion
Here is an example of running FreshRSS behind an Apache reverse proxy (as a subdirectory).
```